utils/loading_cache.hh

```diff
@@ -86,6 +96,7 @@ struct simple_entry_size {
 ///
 /// \tparam Key type of the cache key
 /// \tparam Tp type of the cached value
+/// \tparam SectionHitThreshold number of hits after which a cache item is going to be moved to the privileged cache section.
```

Comment:
I've said this too many times already but the word "after" is ambiguous. It should have been phrased differently.